openage/convert/opus/ogg.pxd

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016-2017 the openage authors. See copying.md for legal info.
+
+cdef extern from "ogg/config_types.h":
+    # TODO hope the best
+    ctypedef short ogg_int16_t
+    ctypedef long ogg_int64_t
```

Comment:
What do we do about that?
libogg has a lot of `#ifdef`s. I didn't find a way to tell cython to just use the definition from the header.
cython is really limiting portability.
can someone maybe come up with a workaround? see the comment on opus.pxd for inspiration.